packages/sources/wbtc-address-set/src/index.ts

```diff
@@ -1,10 +1,13 @@
-import { expose } from '@chainlink/ea-bootstrap'
-import { endpointSelector, makeExecute } from './adapter'
-import * as endpoints from './endpoint'
-import { makeConfig, NAME } from './config'
-import * as types from './endpoint'
+import { expose, ServerInstance } from '@chainlink/external-adapter-framework'
+import { PoRAdapter } from '@chainlink/external-adapter-framework/adapter/por'
+import { config } from './config'
+import { addresses, members } from './endpoint'
 
-const adapterContext = { name: NAME }
+export const adapter = new PoRAdapter({
+  defaultEndpoint: addresses.name,
+  name: 'WBTC',
+  config,
+  endpoints: [addresses, members],
+})
```

Comment:
Since this adapter uses HttpTransports, we should set a reasonable limit. @alejoberardino What do you think about 10 req/s? PoR feeds don't update often so we can even go 1 req/s
```suggestion
export const adapter = new PoRAdapter({
  defaultEndpoint: addresses.name,
  name: 'WBTC',
  config,
  endpoints: [addresses, members],
  rateLimiting: {
    tiers: {
      default: {
        rateLimit1s: 10,
        note: 'Considered unlimited tier, but setting reasonable limits',
      },
    },
  },
})
```

Reply:
I think we can go even lower haha, like once every 10s